Ryujinx.Ava/UI/Windows/MainWindow.axaml.cs

```diff
@@ -327,10 +327,11 @@ public static void UpdateGraphicsConfig()
 
         public void LoadHotKeys()
         {
-            HotKeyManager.SetHotKey(FullscreenHotKey,  new KeyGesture(Key.Enter, KeyModifiers.Alt));
-            HotKeyManager.SetHotKey(FullscreenHotKey2, new KeyGesture(Key.F11));
-            HotKeyManager.SetHotKey(DockToggleHotKey,  new KeyGesture(Key.F9));
-            HotKeyManager.SetHotKey(ExitHotKey,        new KeyGesture(Key.Escape));
+            HotKeyManager.SetHotKey(FullscreenHotKey,      new KeyGesture(Key.Enter, KeyModifiers.Alt));
+            HotKeyManager.SetHotKey(FullscreenHotKey2,     new KeyGesture(Key.F11));
+            HotKeyManager.SetHotKey(FullscreenHotKeyMacOS, new KeyGesture(Key.F, KeyModifiers.Control | KeyModifiers.Meta)); //Mac OS Control+Cmd+F
```

Comment:
I'm not sure if the comment is necessary.

Reply:
```suggestion
            HotKeyManager.SetHotKey(FullscreenHotKeyMacOS, new KeyGesture(Key.F, KeyModifiers.Control | KeyModifiers.Meta));
```
> I'm not sure if the comment is necessary.

Yeah you can remove this comment, it's not.

Reply:
Done